cinema/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=100)
+    description = models.TextField()
+    duration = models.IntegerField()
```

Comment:
For the `duration` field, consider adding a `help_text` to clarify what unit of time you're using (minutes, seconds, etc.).